boards/sim/sim/sim/scripts/Make.defs

```diff
@@ -78,10 +78,6 @@ ifeq ($(CONFIG_SCHED_GCOV_ALL),y)
   ARCHOPTIMIZATION += -fprofile-generate -ftest-coverage
 endif
 
-ifneq ($(CONFIG_PROFILE_ALL)$(CONFIG_SIM_PROFILE),)
-  ARCHOPTIMIZATION += -pg
```

Comment:
could you point out the dup location? I can't find it.

Reply:
arch/sim/src/Makefile:147 There is also this configuration

Reply:
should we remove the dup from arch/sim/src/Makefile?

Reply:
OK